features/cache-frameworks.feature

```diff
@@ -18,4 +18,12 @@ Feature: Focuses projects with pre-compiled cached xcframeworks
     Then tuist warms the cache
     When I remove derived data directory
     When tuist focuses the target App
+    Then I should be able to build for iOS the scheme App
+
+  Scenario: The project is an iOS application with custom configuration and cache flavor (ios_app_with_custom_configuration)
```

Comment:
What's surprising the acceptance test is failing due to 

```bash
$ ./bin/fourier test tuist acceptance features/cache-frameworks.feature:23
... 
Then tuist warms the cache with flavor simulator                                                                         # features/step_definitions/cache.rb:11
      swift run tuist cache warm --path /var/folders/9p/kwbtgrj53cl08_1gwbtm5kvc0000gn/T/d20210108-62798-1ottqko --flavor simulator
      The flavor 'simulator' is missing in your project's configuration. Available cache flavors: development.
```

It seems that a cache config is not loaded from a `Config.swift` manifest file and a default config is used instead. Does anyone have an idea why?

Reply:
Does the test run locally?

Reply:
Apparently, the test is passing after applying the changes 👍

Reply:
Now, when I execute
```bash
./bin/fourier test tuist acceptance features/cache-frameworks.feature:23
```

I got:

```bash
Then tuist warms the cache with simulator profile                                                                         # features/step_definitions/cache.rb:11
      swift run tuist cache warm --path /var/folders/9p/kwbtgrj53cl08_1gwbtm5kvc0000gn/T/d20210127-62892-ngaxvs --profile simulator
      Couldn't find framework 'FrameworkA.framework' in the derived data directory: /Users/djankowski/Library/Developer/Xcode/DerivedData
       (Minitest::Assertion)
      ./features/support/system.rb:9:in `system'
      ./features/step_definitions/cache.rb:12:in `/^tuist warms the cache with ([a-zA-Z]+) profile/'
      features/cache-frameworks.feature:27:in `tuist warms the cache with simulator profile'
```

When I execute the test via terminal myself, it seems that the cache command does not pick up `Config.swift` config: 

```bash
$ swift run --package-path ~/Projects/tuist tuist cache warm --path ~/Projects/tuist/fixtures/ios_app_with_custom_configuration --profile simulator
Selected cache profile 'development' from Tuist's defaults // <--- although the simulator profile is specified in Config
Generating workspace Workspace.xcworkspace
Generating project App
Generating project FrameworkA
Hashing cacheable targets
The target FrameworkA with hash 97c2c7c2b25f87c168306d21b1e83fa7 and type framework is already in the cache. Skipping...
Building cacheable targets
All cacheable targets have been cached successfully as frameworks
```

Any ideas @pepibumur ? 🙏